### ISSUE TITLE
Screen Reader does not announce the state of ‘Anonymous' and ‘Authenticated’ controls.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#113]
User Experience:

Current state of the element is an important information for the user to know and move further with the process, screen reader users will get impacted if state information is not provided via screen reader.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Bottom nav and Select ‘Role selection’ button.

5. Enable screen reader and navigate to all control present inside role selection.

6. Verify the issue.

 

Actual Result:

Screen Reader does not announce the state of ‘Anonymous' and ‘Authenticated’ controls as selected and not selected.

 

Expected Result:

Screen Reader should  announce the state of ‘Anonymous' and ‘Authenticated’ as selected and not selected.

 

Note:

Similar Issue exist with Chrome +JAWS and Chrome + NVDA

 

MAS Reference: 

MAS 4.1.2 - Name, Role, Value (40B)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries
